worker-message-prioritization-distribution/src/test/java/com/github/workerframework/workermessageprioritization/redistribution/TunedTargetQueueIT.java

```diff
@@ -56,17 +58,17 @@
 import java.io.IOException;
 import java.net.InetAddress;
 import java.nio.charset.StandardCharsets;
-import java.util.Collections;
+import java.util.HashMap;
+import java.util.Map;
 import java.util.concurrent.TimeoutException;
 import java.util.stream.IntStream;
 
 @ExtendWith(MockWebServerExtension.class)
 public class TunedTargetQueueIT extends DistributorTestBase {
-
-    final String queueName = "elastic-query-worker";
-    final String stagingQueue1Name = getStagingQueueName(queueName, T1_STAGING_QUEUE_NAME);
-    final String stagingQueue2Name = getStagingQueueName(queueName, T2_STAGING_QUEUE_NAME);
     public static final String MOCK_SERVER_PORT = "CAF_MOCK_SERVER_PORT";
+    public static final String QUEUE_NAME = "elastic-query-worker";
```

Comment:
Try to keep changes focused on the ticket at hand, it makes it more difficult to review the functional (important) changes 😀